### ISSUE TITLE
default setings

### ISSUE THREAD

**Issue by Pushplaybang** [chrismbeckett/meteor-toastr#6]
Is there a way to set default options for the lib?  should it be it be initialised the same way as the standard library?

**Comment by chrismbeckett** [comment]
Yes, the API and settings are exactly the same as the original library.
